docs/models/utils/tag.md

```diff
@@ -0,0 +1,16 @@
+# Tags
+
+Tags are text labels which can be applied to objects. Each tag has a label and
+a URL-friendly slug. The slug is generated automatically and makes tags easier
+to work with as URL parameters. Objects can be filtered by tags.
+
+For each tag that you create, the following properties are to be provided (some
+are however optional):
+
+  * `Name`: a human readable name attached to a tag.
+  * `Slug`: a unique configuration and URL friendly name; most of the time it
```

Comment:
Suggest "...and URL-friendly name. Usually automatically generated from the tag's name"